### ISSUE TITLE
Not using TensorFlow?

### ISSUE THREAD

**Issue by GabrielLin** [rickyhan/SimGAN-Captcha#3]
It seems that tensorflow is used for calculating two loss function. It is not used during training. Is it possible to use numpy to calculate those loss function. Then, Theano backend can be used for Keras. Thanks.
